src/xdocs/config_javadoc.xml

```diff
@@ -1213,6 +1213,138 @@ public int checkReturnTag(final int aTagIndex,
       </subsection>
     </section>
 
+    <section name="JavadocSummaryLocation">
+      <p>Since Checkstyle 8.26</p>
+      <subsection name="Description" id="JavadocSummaryLocation_Description">
+        <p>
+          Checks that <a
+          href="https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#firstsentence">
+          Javadoc summary sentence</a> is in the same position for all javadoc in the project.
+        </p>
+        <p>
+          It is possible to enforce two different styles:
+        </p>
+        <ul>
+          <li>
+            {@code First line} - javadoc summary starts on the first line:
+            <source>
+&#47;** Summary text.
+* More details.
+*&#47;
+public void method();
+            </source>
+          </li>
+          <li>
+            {@code Second line} - javadoc summary starts on the second line:
+            <source>
+&#47;**
+* Summary text.
+* More details.
+*&#47;
+public void method();
+            </source>
+          </li>
+        </ul>
+      </subsection>
+
+      <subsection name="Properties" id="JavadocSummaryLocation_Properties">
+        <table>
+          <tr>
+            <th>name</th>
+            <th>description</th>
+            <th>type</th>
+            <th>default value</th>
+            <th>since</th>
+          </tr>
+          <tr>
+            <td>violateExecutionOnNonTightHtml</td>
+            <td>
+              Control when to print violations if the Javadoc being examined by this check
+              violates the tight html rules defined at
+              <a href="writingjavadocchecks.html#Tight-HTML_rules">Tight-HTML Rules</a>.</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>false</code></td>
+            <td>8.3</td>
+          </tr>
+          <tr>
+            <td>location</td>
+            <td>Specify the policy on placement of the javadoc summary.</td>
+            <td>
+              <a href="property_types.html#javadocSummaryLocation">Javadoc Summary Location</a>
+            </td>
+            <td><code>second_line</code></td>
+            <td>8.25</td>
+          </tr>
+        </table>
+      </subsection>
+
+      <subsection name="Examples" id="JavadocSummaryLocation_Examples">
+        <p>
+          By default Check validate that the summary is located on the second line:
+        </p>
+        <source>
+&lt;module name=&quot;JavadocSummaryLocationCheck&quot;/&gt;
+        </source>
+        <p>
+          To ensure that summary is located on the first line:
+        </p>
+        <source>
+&lt;module name=&quot;JavadocSummaryLocationCheck&quot;&gt;
+  &lt;property name=&quot;location&quot; value=&quot;FIRST_LINE&quot;/&gt;
+&lt;/module&gt;
```

Comment:
we need to share examples of code and where violation will be (as trailing comment, kind of copy of what you have in Inputs). Users love examples.

Reply:
done